drivers/soundwire/mipi_disco.c

```diff
@@ -268,7 +268,7 @@ static int sdw_slave_read_dpn(struct sdw_slave *slave,
 					sizeof(*dpn[i].ch_combinations),
 					GFP_KERNEL);
 			if (!dpn[i].ch_combinations) {
-				fwnode_handle_put(node);						
+				fwnode_handle_put(node);
```

Comment:
What has happened here?

Reply:
probably I removed a trailing whitespace that was there before...

Reply:
indeed, I need to rebase all this after correcting the initial mistake.

Reply:
fixed